External/FEXCore/Source/Interface/Context/Context.h

```diff
@@ -315,6 +328,9 @@ namespace FEXCore::Context {
     void NotifyPause();
 
     void AddBlockMapping(FEXCore::Core::InternalThreadState *Thread, uint64_t Address, void *Ptr, uint64_t Start, uint64_t Length);
+    void SMCMarkReadOnly(uint64_t Start, uint64_t Length);
```

Comment:
Could you add a docstring explaining what this function does?

Reply:
Done